drone-webdriver/src/main/java/org/jboss/arquillian/drone/webdriver/factory/HtmlUnitDriverFactory.java

```diff
@@ -86,4 +101,22 @@ public Capabilities getCapabilities(WebDriverConfiguration configuration, boolea
     protected String getDriverReadableName() {
         return BROWSER_CAPABILITIES;
     }
+
+    private void setClientOptions(WebClient webClient, String htmlUnitWebClientOptions) {
+        final WebClientOptions webClientOptions = webClient.getOptions();
+
+        Map<String, String> clientOptions = new LinkedHashMap<>();
+        final String[] options = htmlUnitWebClientOptions.split(",");
```

Comment:
what if we want to set an array of strings as a value - eg: http://htmlunit.sourceforge.net/apidocs/com/gargoylesoftware/htmlunit/WebClientOptions.html#setSSLClientProtocols-java.lang.String:A-
correct me if my question is nonsense...